bin/runtime/src/lib.rs

```diff
@@ -1011,6 +1001,12 @@ impl_runtime_apis! {
         fn next_session_finality_version() -> FinalityVersion {
             Aleph::next_session_finality_version()
         }
+
+        fn session_committee(
+            session: SessionIndex,
```

Comment:
Is this argument necessary? Aren't we interested only in current's session committee or at some given block where we are manually computing it from some session number?

Reply:
thats the whole point of this function to be able to compute committee for **any** session in the current era